easybuild/easyconfigs/p/PLAST/PLAST-2.3.1-foss-2016a-Java-1.8.0_92.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'PLAST'
+version = '2.3.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = "http://www.irisa.fr/symbiose/projects/plast/" 
+description = """PLAST is a parallel alignment search tool for comparing large protein banks"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/PLAST-software/plast-library/releases/download/v%(version)s/']
+sources=['%(namelower)s_source_v%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.5.2')
```

Comment:
@RvDijk missing `,` at the end of this line, leading to a parse error: `TypeError: 'tuple' object is not callable`